packages/loader/container-definitions/src/error.ts

```diff
@@ -20,7 +20,7 @@ export enum ContainerErrorType {
     /**
      * Data loss error detected by Container / DeltaManager. Likely points to storage issue.
      */
-    dataCorruption = "dataCorruption",
```

Comment:
can we keep the enum symbol unmodified, as otherwise this is technically a breaking change. probably no one is using it, but never know.

Reply:
We absolutely _can_, but I did think it would be beneficial to normalize the names with the other enums (`"genericError"`, and `"throttlingError"`) in addition to preserving compatibility with what was originally surfaced by `DataCorruptionError` instances.